invenio_administration/assets/semantic-ui/js/invenio_administration/src/search/search.js

```diff
@@ -32,6 +33,9 @@ const actions = JSON.parse(domContainer.dataset.actions);
 const apiEndpoint = _get(domContainer.dataset, "apiEndpoint");
 const idKeyPath = JSON.parse(_get(domContainer.dataset, "pidPath", "pid"));
 const listUIEndpoint = domContainer.dataset.listEndpoint;
+const resourceSchema = JSON.parse(domContainer.dataset.resourceSchema);
+
+enrichResourceSchema(resourceSchema);
```

Comment:
why not using Formatter directly in the component that needs it? We could discuss in person